problem-30.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+// Uses structures to show names, roll numbers and percentages of 5 students but stores and reads content from binary file
+
+typedef struct
+{
+    char *name;
+    int roll_number;
+    float percentage;
+} Student;
+
+int main()
+{
+    FILE *file;
+    Student data[5];

```

Comment:
1. Same as above.
1. Better to use name `students` instead of `data` (same feedback for above code)